tests/api_tests/users/test_views.py

```diff
@@ -380,3 +372,17 @@ def test_get_404_path_nodes_user_id_no_user(self):
         url = "/{}nodes/{}/".format(API_BASE, self.user_one._id)
         res = self.app.get(url, expect_errors=True)
         assert_equal(res.status_code, 404)
+
+
+class TestDeactivatedUser(ApiTestCase):
+
+    def setUp(self):
+        super(TestDeactivatedUser, self).setUp()
+        self.user = AuthUserFactory()
+        self.user.is_disabled = True
+        self.user.save()
+
+    def test_return_deactivated_user(self):
+        self.url = '/{}users/{}/'.format(API_BASE, self.user._id)
```

Comment:
There's no need to assign this as an instance variable.

I'll change this when I merge.